app/controllers/talking_stick/participants_controller.rb

```diff
@@ -42,7 +41,7 @@ def create
     # PATCH/PUT /participants/1
     def update
       if @participant.update(participant_params)
-        redirect_to [@room, @participant], notice: 'Participant was successfully updated.'
+        redirect_to talking_stick.room_path(@room), notice: 'Participant was successfully updated.'
```

Comment:
You're right, it is. Originally, I thought it made more sense this way, but as I think about it now, a CRUD for Participants really doesn't need to exist (except perhaps as a debugging feature). You can't manually add participants - it's only supposed to happen via the JSON API.